hazelcast/hatch.toml

```diff
@@ -2,9 +2,10 @@
 
 [[envs.default.matrix]]
 python = ["3.11"]
-version = ["4.0"]
+version = ["5.0"]
```

Comment:
Same here, we could keep the 4.0 env and just make the additions to 5.0 versions

Reply:
Same comment as above - do we need to support the 4.0 version indefinitely? or should we move to the stable major version. I believe they are also working on releasing version 6.